tests/unit/js/test.ol.js

```diff
@@ -10,19 +10,27 @@ beforeEach(() => {
     sandbox.stub(global, '$').callsFake(jquery);
 });
 
-test('initReadingListFeature binds events to HTML added after initialisation', () => {
-    const spy = sinon.spy();
-    initReadingListFeature(() => spy);
-    $(document.body).html(`<div>
-    <div class="dropclick">test</div>
-    <a class="add-to-list"></a>
-</div>
-    `);
-    // trigger a click event
-    $('.dropclick').trigger('click');
-    // check the event handler was called
-    expect(spy.callCount).toBe(1);
-    $('.dropclick').trigger('click');
-    // check the dropclick event handler was called
-    expect(spy.callCount).toBe(2);
+describe('initReadingList', () => {
+    test('dropdown changes arrow direction on click', () => {
+        const clock = sinon.useFakeTimers();
+        initReadingListFeature();
+        $(document.body).html(`
+            <a href="javascript:;" class="dropclick dropclick-unactivated">
+                <div class="arrow arrow-unactivated"></div>
+            </a>
+        `);
+
+        const $dropclick = $('.dropclick');
+        const $arrow = $dropclick.find('.arrow');
+
+        for (let i = 0; i < 2; i++) {
+            $dropclick.trigger('click');
+            clock.next(); // need to step forward because using debounce
```

Comment:
Why not stub debounce here with a function that executes right away? e.g. 

```
import utils from './nonjquery_utils.js';
fakeDebounce = (fn)=>fn(); 
sinon.stub( utils, 'debounce' ).callsFake( fakeDebounce )
```

I prefer to test dedicated units and this this unit test is also testing debounce which you already have good coverage for.

Reply:
DONE